flit/install.py

```diff
@@ -201,6 +201,7 @@ def install_data_dir(self, target_data_dir):
             rel_path = os.path.relpath(src_path, self.ini_info.data_directory)
             dst_path = os.path.join(target_data_dir, rel_path)
             os.makedirs(os.path.dirname(dst_path), exist_ok=True)
+            pathlib.Path(dst_path, missing_ok=True)
```

Comment:
```suggestion
            pathlib.Path(dst_path).unlink(missing_ok=True)
```

I think this is what we need.

There's a bug where `Path` currently accepts and ignores keyword arguments, so this line as is isn't an error. It looks like this is being fixed, with a warning from 3.12 and an error from 3.14. https://github.com/python/cpython/pull/19632